cli/src/commands/deploy.ts

```diff
@@ -194,7 +198,10 @@ export default class Deploy extends Command {
   }
 
   public async run() {
-    const { flags: { 'app-name': givenAppName, env: givenEnv } } = this.parse(Deploy);
+    const { flags: { 'app-name': givenAppName, env: givenEnv, 'new-app': forceNewApp } } = this.parse(Deploy);
```

Comment:
It seems OClif can do this.  Perhaps use `exclusive` (https://oclif.io/docs/flags)?